pkg/image/controller/imagestream_controller.go

```diff
@@ -116,14 +116,21 @@ func (c *ImageStreamController) processNextWorkItem() bool {
 	}
 	defer c.queue.Done(key)
 
-	stream, err := c.getByKey(key.(string))
+	namespace, name, err := cache.SplitMetaNamespaceKey(key.(string))
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("Error syncing image stream: %v", err))
+		glog.V(2).Infof("Unable to split namespace key for key %q: %v", key, err)
```

Comment:
`utilruntime.HandleError`, then forget and drop the key.  The key is never going to get fixed.